core/block_svg.js

```diff
@@ -634,11 +634,12 @@ Blockly.BlockSvg.prototype.showHelp_ = function() {
 };
 
 /**
- * Show the context menu for this block.
- * @param {!Event} e Mouse event.
- * @private
+ * Generate the context menu for this block.
+ * @protected
+ * @returns {Array} Context menu options
```

Comment:
The JSDoc doesn't match the return value.  Either change this annotation to
```
{!Array.<!Object>|undefined}
```
or (in my opinion this would be better) change the annotation to 
```
{Array.<!Object>}
```
and change the return on 644 to be
```
return null;
```

Also, drop the blank line on 642.

Otherwise, looks good to me.

Reply:
This PR looks like it was already merged and closed; would you like me to make a new PR with those changes, or?

Reply:
New PR, please.

Reply:
I've just folded these minor nits into PR #2598 which also edits this file.